values.yaml

```
@@ -263,6 +263,26 @@ server:
     secretName: null
     secretKey: null
 
+  # exposeGossipAndRPCPorts exposes the servers' gossip and RPC ports as
+  # hostPorts. To enable a client agent outside of the k8s cluster to join the
+  # datacenter, you would need to enable server.exposeGossipAndRPCPorts,
+  # client.exposeGossipPorts, and set server.serflan.port to a port not being
+  # used on the host. Since client.exposeGossipPorts uses the hostPort 8301,
+  # server.serflan.port must be set to something other than 8301.
+  exposeGossipAndRPCPorts: false
```

Excellent comment

```suggestion
  # Exposes the servers' gossip and RPC ports as
  # hostPorts. To enable a client agent outside of the k8s cluster to join the
  # datacenter, you would need to enable `server.exposeGossipAndRPCPorts`,
  # `client.exposeGossipPorts`, and set `server.serflan.port `to a port not being
  # used on the host. Since `client.exposeGossipPorts` uses the hostPort 8301,
  # `server.serflan.port` must be set to something other than 8301.
  exposeGossipAndRPCPorts: false
```

nit: the style for these comments will change once https://github.com/hashicorp/consul-helm/pull/727 is merged. 

The change is that we won't use the name of the key at the start of the description and also we can use backticks in the description to get code blocks on the markdown.

---

Yup, thanks!